daemon/cmd/daemon_main.go

```diff
@@ -1564,6 +1565,42 @@ func initClockSourceOption() {
 	}
 }
 
+func checkHostFirewallWithEgressLB() {
+	// Egress LB is enabled in datapath under condition:
+	// ENABLE_SERVICES && (!ENABLE_HOST_SERVICES_FULL || \
+	//                    (ENABLE_EXTERNAL_IP && !BPF_HAVE_NETNS_COOKIE))
+	// We can't enable both egress LB and host firewall on kernels <4.14 due to
+	// the verifier complexity limit, at 96k instructions.
+	netnsCookieSupport := false
+	pm := probes.NewProbeManager()
+	if h1, h2 := pm.GetHelpers("cgroup_sock_addr"), pm.GetHelpers("cgroup_sock"); h1 != nil && h2 != nil {
```

Comment:
You can leave the code as it is but just a FYI you can access nil maps. https://play.golang.org/p/t__FGNkDvCx